source/cpu/se_s390x.go

```diff
@@ -0,0 +1,38 @@
+//go:build s390x
+// +build s390x
+
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cpu
+
+import "os"
+
+// This assumes the `host-sys` mount, which provides access to the host sysfs.
+const protVirtHost = "/host-sys/firmware/uv/prot_virt_host"
```

Comment:
Should we use `source.SysfsDir` here?

Reply:
Uh-oh, yes, definitely. Thanks @Ethyling!

Reply:
We should. Tested & done.